code/datums/components/reskin.dm

```diff
@@ -1890,7 +1890,7 @@ GLOBAL_LIST_EMPTY(reskin_list)
 	skin = "P-14"
 	name = "P-14"
 	desc = "A bolt-action rifle. Formerly chambered in 14mm, until they missed a payment to the 14mm board. The 'P' remains a mystery."
-	icon = 'modular_coyote/icons/objects/rifles.dmi'
+	icon = 'icons/fallout/objects/guns/longguns.dmi '
```

Comment:
oh now that im on desktop i see the problem. theres an extra space at the end of the filename here